src/services/AssetService.ts

```diff
@@ -6,17 +7,43 @@ import { FlowComponents } from '../store/helpers';
 export class Assets {
     private endpoint: string;
     private localStorage: boolean;
-    protected items: SearchResult[] = [];
     protected nameProperty: string;
     protected idProperty: string;
 
+    // TODO: consider redux backed by local storage for this
```

Comment:
Sorry, can you clarify the consideration here? All of this could exist in redux with checks to determine when to persist/hydrate asset state, right?

Reply:
Nice, perhaps asset persistence is gated by an env variable?

Reply:
The more I think about this the more I think we should remove the redux requirement for all of this since it isn't a necessary piece in the embedded case. If nothing is rendering based on it, I don't see the value in putting it in redux and it should probably be a goal that nothing is rendering off the assets from the `AssetService` since they are remote in the embed scenario. If that's the case then it would look more like:

#### Local Dev
 * Faux json asset endpoints
 * Assets merged with items stored in browser local storage

#### Netlify preview
 * Faux json asset endpoints
 * Assets merged with items encountered in loading of flow definitions stored in memory

#### Embed
 * All assets fetched on demand from asset endpoints, no local cache or storage

This means removing `flows` and `groups` from redux and fixing the dependencies on those to be on demand via the `AssetService`.

Reply:
That said, local dev and netlify case might be implemented the same with the only difference being that the local dev case would just hydrate from local storage into memory and write it out as needed.